tests/frameworks/test_huggingface.py

```diff
@@ -100,33 +100,35 @@ def args():
         "foo",
         evaluation_strategy="epoch",
         num_train_epochs=2,
-        save_strategy="epoch",
+        save_strategy="no",
+        logging_strategy="no",
```

Comment:
[F] [Defaults to steps](https://huggingface.co/docs/transformers/v4.37.2/en/main_classes/trainer#transformers.TrainingArguments.logging_strategy). Which I think means that we would be logging multiple times per epoch... which is unexpected.